src/components/pool_flux_chart.vue

```diff
@@ -130,6 +132,18 @@ export default {
           // pools and fluxes of water
           this.volume = data[0];
 
+          // define custom number formatting for xAxis linear scale
+          let formatBillion = function(x) { return self.d3.format(".1f")(x / 1e9) + "B"; },
+              formatMillion = function(x) { return self.d3.format(".0f")(x / 1e6) + "M"; },
+              formatThousand = function(x) { return self.d3.format(".0f")(x / 1e3) + "k"; };
+
+          this.customNumberFormat = function(x) {
```

Comment:
Hmm my thought was to have the common axis values be formatted the same way between the two scales, so that both have 100M and 1B

I think 1000M instead of 1B would be confusing

I tend to find axes that have .1M instead of 100K more confusing than ones that mix suffixes, but that may just be me

Reply:
can you post on the viz channel or chat to get some feedback from others on this?